utils/emulator_utils.py

```diff
@@ -1,6 +1,6 @@
 """ Utilities for adding a template emulator for a new IBEX device"""
 from templates.paths import EMULATOR_TEMPLATE
-from file_system_utils import copy_tree
+from .file_system_utils import copy_tree
```

Comment:
I think we should use absolute imports here and everywhere else that imports relatively (ie. `from common.file_system_utils import copy_tree`) as they are less fragile and read nicer - this will probably break running the device generator in python 2, but im not sure we should care?

I can do this if you want?

Reply:
also means we can get rid of that horrible check in `command_line_utils.py` to see if the user is running python 2